code/Game.cs

```diff
@@ -5,8 +5,13 @@
 global using System;
 global using System.Collections.Generic;
 global using System.ComponentModel;
+global using System.Threading.Tasks;
+global using System.ComponentModel.DataAnnotations;
 global using System.Linq;
 
+using Sports.PartySystem;
```

Comment:
don't think we need this using anymore. Gurke removed it. but your PR seems to add it again. same for System.Threading.Tasks and System.ComponentModel.DataAnnotations.